### ISSUE TITLE
Remove aria-label attribute causing accessibility issues in lighthouse report

### ISSUE THREAD

**Issue by mgayaud** [calibreapp/react-live-chat-loader#116]
This was observed on the live chat loader using it for Intercom. The div element wrapping the loader has an aria-label attribute causing an accessibility issue on lighthouse. Removing the aria-label by hand fixed the issue for me.

## What problem would this feature solve?
The issue on the lighthouse report (accessibility) would be solved.

## Describe the solution you’d like to see
Remove the aria-label="Load Chat" attribute.

## Describe alternatives you’ve considered
Replace the aria-label attribute by another one not causing a lighthouse error.

## Additional context
Here are some screenshots showing the issue

![Capture d’écran 2022-01-05 à 14 57 29](https://user-images.githubusercontent.com/35394427/148231412-eaa591a9-6cb2-443d-845c-5b437c16390f.png)
![Capture d’écran 2022-01-05 à 10 42 07](https://user-images.githubusercontent.com/35394427/148231410-b35ee500-192f-4f46-994c-9a759ea7e61c.png)

**Comment by mikedijkstra** [comment]
Hi @mgayaud — What version are you using?

In October we released an update which added `role=button` to these divs to fix this issue.

Please try updating to the latest version. I'll close this for now but feel free to re-open if the issue remains.

**Comment by mgayaud** [comment]
Thank you @mikedijkstra for your quick reply !

I'm currently using the 2.7.2 version as indicated in my `package.json`
`"react-live-chat-loader": "^2.7.2",`

I used the `npm install --save react-live-chat-loader` to install the module.

Looking at the file at
`/node_modules/react-live-chat-loader/dist/components/Intercom/index.js`
and `/node_modules/react-live-chat-loader/module/components/Intercom/index.js`
I can only see the following lines :
```javascript
  }, /*#__PURE__*/_react.default.createElement("div", {
    "aria-label": "Load Chat",
    "aria-busy": "true",
    "aria-live": "polite",
[...] })
```

I can see the `role=button` attribute on the repository source code but not in the file I have installed.

Do you have any idea why this issue is happening ?

**Comment by mikedijkstra** [comment]
Hi @mgayaud,

It looks like the last publish didn't include the updated build files. I've just published 2.7.3 which should include the correct build! Please try updating again.

Sorry for the confusion!

**Comment by mgayaud** [comment]
Thank you @mikedijkstra, the update worked !
